hyp3_gamma/__main__.py

```diff
@@ -33,10 +34,20 @@ def main():
     )
 
 
+def write_earthdata_creds(username, password):
```

Comment:
I slightly prefer to avoid abbreviations, so I'd suggest `credentials` over `creds`, but that's not a sticking point